gradle/libs.versions.toml

```diff
@@ -66,6 +69,9 @@ managed-r2dbc-io-asyncer-mysql = { module = "io.asyncer:r2dbc-mysql", version.re
 managed-r2dbc-mssql = { module = "io.r2dbc:r2dbc-mssql", version.ref = "managed-r2dbc-mssql" }
 managed-r2dbc-postgresql = { module = "org.postgresql:r2dbc-postgresql", version.ref = "managed-r2dbc-postgresql" }
 
+# Reactor Netty Dependencies
```

Comment:
It turns out maria-r2dbc 1.2.1 version removed reactor-netty dependency and added only reactor-netty-core. This got reactor-netty-http removed and I noticed Postgres r2dbc tests were failing and hanging indefinitely. 
Adding this dependency resolved the issue with failing Postgres tests. So, apparently this project needed reactor-netty-http dependency that it was getting via maria r2dbc dependency but probably we were not aware. We can leave it like this, or add whole reactor-netty dependency, or else we could have version diffs between catalog and reactor-netty-core coming from maria r2dbc dependency.